src/test/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineStepApiLegacyTest.java

```diff
@@ -214,6 +214,8 @@ public void getAllStepsReturnsStepsForComplexParallelBranches() throws Exception
         assertThat(steps.get(8).getTitle(), is("Print Message"));
         assertThat(steps.get(9).getName(), is("In stage Nested 2 - 2 within Branch C"));
         assertThat(steps.get(9).getTitle(), is("Print Message"));
+        assertThat(steps.get(10).getName(), is("Get contextual object from internal APIs"));
+        assertThat(steps.get(10).getTitle(), is(""));
```

Comment:
Being consistent with other code in this file, which uses `is("")` rather than `isEmpty()` or the like.